sites/markdown-editor/src/App.jsx

```diff
@@ -79,8 +90,9 @@ export default withStyles(() => ({
     alignItems: 'stretch',
     minWidth: 0,
     minHeight: 0,
-    overflow: 'hidden',
+    // overflow: 'hidden',
```

Comment:
So this is the tricky and hacky part: because I wanted to have the webview on top of the components, I didn't want it to scroll. So I removed all the `height: 100%` and `overflow: hidden` so that the body grows with the content.